.github/labeler.yml

```diff
@@ -170,3 +173,135 @@ module:sql:
   - changed-files:
       - any-glob-to-any-file:
           - solr/modules/sql/**
+
+# Any test source files
+tests:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/src/test/**'
+          - '**/src/test-files/**'
+
+# Add category labels based on java pacakges
+# In essence, for a package org.apache.solr.foo[.xxx], assign label cat:foo
+cat:analysis:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/analysis/**'
+
+cat:cli:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/cli/**'
+
+cat:cloud:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/cloud/**'
+
+cat:cluster:
```

Comment:
I don't think this is really distinct from SolrCloud

Reply:
Probably not, will consolidate with 'cloud'